tests/relay_integration/test_integration.py

```diff
@@ -196,3 +201,11 @@ def test_transaction(self):
                     "total.time": {"value": pytest.approx(1050)},
                 }
             }
+
+            assert [span["exclusive-time"] for span in raw_event["spans"]] == [
```

Comment:
Could you double-check the spelling here? Looks like this should be `exclusive_time`.

Reply:
It's in kebab-case afaik.

Reply:
This is from the event payload, right? Payload should be all snake_case. See the parsing test in Relay: https://github.com/getsentry/relay/pull/1061/files#diff-27bbe8166c22c9062d109bab3ce15e368ab09735ec71fd1d7bb5321744c3e07cR66